### PR TITLE
doc(bigtable): revamp examples README

### LINKED ISSUE #6818: Docs: Update Bigtable examples README.md
It is outdated and instructs users to use targets that no longer exist. Some links point to the wrong place.

We could also use this opportunity to make the examples/samples README's conform across services.

### PR DESCRIPTION
**Pull request by dbolduc**
Fixes #6818

---
This change is [Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/6839)

### CODE REVIEW

File: google/cloud/bigtable/examples/README.md [hunk 1]
```diff
@@ -1,208 +1,143 @@
...
+View the [Hello Table Admin][table_admin_code] example to see sample usage of table
+administration of the Bigtable client library. More details on this sample code can be found [here](https://googleapis.dev/cpp/google-cloud-bigtable/latest/bigtable-hello-table-admin.html).
 
-#### Run instance admin samples
+### Running `TableAdmin` Samples
 
-After configuring gRPC, you can run the examples using:
+| Target                                   | Description                         |
+| ---------------------------------------- | ----------------------------------- |
+| `./bigtable_hello_table_admin`           | Demonstration of basic operations   |
```
Comment (coryan) [inline google/cloud/bigtable/examples/README.md:98]: s/Demonstration/Demonstrates/ maybe?

File: google/cloud/bigtable/examples/README.md [hunk 2]
```diff
@@ -1,208 +1,143 @@
...
-$ export TABLE_ID=... # Cloud Bigtable table ID
-$ ./bigtable_samples run ${PROJECT_ID} ${INSTANCE_ID} ${TABLE_ID}
+$ ./read_snippets read-row <project-id> <instance-id> <table-id> <row-key>
 ```
 
-### Reading and Writing Operations
+## Credentials
 
-This sample showcases reading and writing operations:
+The following samples demonstrate use of [Authentication](https://cloud.google.com/bigtable/docs/authentication) and [Access Control]((https://cloud.google.com/bigtable/docs/access-control)) for Bigtable. More details on the samples can be found [here](https://googleapis.dev/cpp/google-cloud-bigtable/latest/bigtable-samples-grpc-credentials.html).
```
Comment (coryan) [inline google/cloud/bigtable/examples/README.md:127]: For long links like these consider using [reference style links](https://www.markdownguide.org/basic-syntax/#reference-style-links)

**Review by coryan — APPROVED** [review]
A few suggestions below.

**Review by dbolduc — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/6839)* status: 0 of 2 files reviewed, 2 unresolved discussions (waiting on @coryan)

___
*[google/cloud/bigtable/examples/README.md, line 98 at r2](https://reviewable.io/reviews/googleapis/google-cloud-cpp/6839#-Mco_BznYvFo4poPXxeF-r2-98:-Mcr9GA-6tXfV8HKOlFh:b-7r8mrp) ([raw file](https://github.com/googleapis/google-cloud-cpp/blob/96ab255ca97c0696387bf9784f638dce4b9084bf/google/cloud/bigtable/examples/README.md#L98)):*
<details><summary><i>Previously, coryan (Carlos O'Ryan) wrote…</i></summary><blockquote>

s/Demonstration/Demonstrates/ maybe?
</blockquote></details>

I think what I have is clunky. But it is self-consistent. The descriptions all say what the code is (not what the code does).

I am going to leave it as it is, but perhaps this is evidence that a table was not the best data structure
___
*[google/cloud/bigtable/examples/README.md, line 127 at r2](https://reviewable.io/reviews/googleapis/google-cloud-cpp/6839#-Mco_BznYvFo4poPXxeF-r2-127:-McrCzm03SFvgA4dOvNx:bt2vr1b) ([raw file](https://github.com/googleapis/google-cloud-cpp/blob/96ab255ca97c0696387bf9784f638dce4b9084bf/google/cloud/bigtable/examples/README.md#L127)):*
<details><summary><i>Previously, coryan (Carlos O'Ryan) wrote…</i></summary><blockquote>

For long links like these consider using [reference style links](https://www.markdownguide.org/basic-syntax/#reference-style-links)
</blockquote></details>

Fixed.

**Pull request merged**